include/lsst/meas/base/ShapeUtilities.h

```diff
@@ -130,7 +130,8 @@ class ShapeResultKey : public afw::table::FunctorKey<ShapeResult>  {
         afw::table::Schema & schema,
         std::string const & name,
         std::string const & doc,
-        UncertaintyEnum uncertainty
+        UncertaintyEnum uncertainty,
+        afw::table::CoordinateType coordType=afw::table::CoordinateType::PIXEL
```

Comment:
You use this like an `enum class` here but like a regular `enum` elsewhere.  Unless I've caught you in the midst of changing it, one of these is wrong.